12-target-details.tex

```diff
@@ -5,6 +5,144 @@ \section{Javascript}
 \label{target-javascript}
 \state{NoContent}
 
+\subsection{Getting started with Javascript}
+\label{target-javascript-getting-started}
+
+Developing Javascript applications is really powerful with Haxe. Let's see our first sample.
```

Comment:
"is rellay powerful" is sort of out of context. Why should the building be powerful? It can be easier, Haxe can be apowerful tool...etc. also "Let's lookat our first..."